lms/services/event.py

```diff
@@ -38,6 +43,19 @@ def insert_event(self, event: BaseEvent):
 
         return event
 
+    @staticmethod
+    def queue_event(event: BaseEvent) -> None:
```

Comment:
What is the convention around typing of functions that don't return anything. In the frontend for example we would always omit the return type for functions that don't return anything, except in contexts where that is not possible.

Reply:
I'm not entirely sure we have a in-house convention. We currently don't use any type checker so this is more about documenting that it doesn't return anything.